spec/board_spec.rb

```diff
@@ -0,0 +1,68 @@
+# spec :board_spec.rb
+require './lib/board.rb'
+
+RSpec.describe Board do
+  let(:board) { Board.new }
+  describe '.valid?' do
+    context 'when user gives input' do
+      it 'returns true if this input is in range of 1..9 and this input is not taken' do
+        expect(board.valid?([1, 2])).to eq true
+      end
+
+      it 'returns false if this input is taken' do
+        board.update([1, 2])
+        expect(board.valid?([1, 2])).to eq false
+      end
+
+      it 'returns false if this input is not in range of 1..9' do
+        expect(board.valid?([1, 12])).to eq false
+      end
+    end
+  end
+
+  describe '.update' do
+    context 'when input is valid' do
+      it 'push input to board' do
+        board.update([1, 2])
+        expect(board.board.length).to eq 2
+      end
+    end
+  end
+
+  describe '.win?' do
+    context 'when win condition is met' do
+      it 'returns true' do
```

Comment:
- [ ] check if specs cover positive and negative scenarios. Let's add check for `win?` returning false.